examples/kitchensink/src/config/environments/development.ts

```diff
@@ -19,6 +19,10 @@ export default async (): Promise<Config> => ({
     gzip: true,
     indexCatchAll: true,
     serveStatic: true,
+    bearer: {
+      header: 'x-axccess-token',
```

Comment:
is there a small typo here? (`x-axccess` vs `x-access`) ?

Reply:
Also - should this field be default? (vs `Authorization: Bearer <token>` and the field is `Authorization`)

Reply:
Yes! That is very nasty typo! My bad 😢 Well spotted!

It can also be another field, as long as it's configurable. Making `Authorization` the default is fine by me, but I'm noet sure if `permit` is able to strip the `Bearer` part from the header value.

Reply:
Yes I think `Authorization` is preferable, unless you saw that `X-Access-Token` is a new standard (I googled a bit and didn't arrive at a conclusion)